okapi/core.php

```diff
@@ -1206,6 +1206,83 @@ public static function get_node_code()
     }
 
     /**
+     * Return the recommended okapi_base_url.
+     *
+     * This is the URL which we want all *new* client applications to use.
+     * OKAPI will suggest URLs with this prefix in various context, e.g. in all
+     * the dynamically generated docs.
+     *
+     * Also see `get_allowed_base_urls` method.
+     */
+    public static function get_recommended_base_url()
+    {
+        return Settings::get('SITE_URL')."okapi/";
```

Comment:
Oops, you're right: https://www.opencaching.de/okapi/services/apisrv/installation

I thought that SITE_URL was not allowed to change from http://www.opencaching.de, but this was not enforced (because #343 was not done before starting https). Then, OKAPI will behave much better with https than I thought.